src/resources/account/account.router.js

```diff
@@ -0,0 +1,21 @@
+const { Router } = require('express');
+const {
+  createAccount, deleteAccount, getAccountById, updateAccount,
+} = require('./account.service');
+
+const router = Router();
+// eslint-disable-next-line no-unused-vars
+const httpError = require('../../tools/httpError');
+// eslint-disable-next-line no-empty-pattern
```

Comment:
Please configure your eslint configuration. Thats not good to use eslint and ignore the rules

Reply:
@devbarden  Thank you for your feedback! I have fixed it.